uint8 LOW   = 0   # 低空 testhigh
uint8 HIGH  = 1   # 高空

uint8 to_drone_type